pkg/agent/openflow/fields.go

```diff
@@ -103,16 +118,20 @@ var (
 	NodePortAddressField = binding.NewRegField(4, 19, 19, "NodePortAddress")
 	// ToNodePortAddressRegMark marks that the Service type as NodePort.
 	ToNodePortAddressRegMark = binding.NewRegMark(NodePortAddressField, 0b1)
-	// reg4[20]: Field to mark that whether the packet of Service NodePort/LoadBalancer from gateway requires SNAT.
-	ServiceSNATField = binding.NewRegField(4, 20, 20, "ServiceSNAT")
-	// ServiceNeedSNATRegMark marks that the packet of Service NodePort/LoadBalancer requires SNAT.
-	ServiceNeedSNATRegMark = binding.NewRegMark(ServiceSNATField, 0b1)
 	// reg4[16..19]: Field to store the union value of Endpoint state and the mark of whether Service type is NodePort.
 	NodePortUnionField = binding.NewRegField(4, 16, 19, "NodePortUnion")
 	// reg4[21]: Mark to indicate the packet is from local AntreaFlexibleIPAM Pod.
 	// NotAntreaFlexibleIPAMRegMark will be used with RewriteMACRegMark, thus the reg id must not be same due to the limitation of ofnet library.
 	AntreaFlexibleIPAMRegMark    = binding.NewOneBitRegMark(4, 21, "AntreaFlexibleIPAM")
 	NotAntreaFlexibleIPAMRegMark = binding.NewOneBitZeroRegMark(4, 21, "AntreaFlexibleIPAM")
+	// reg4[22..23]: Field to store the state of a connection of Service NodePort/LoadBalancer from gateway which
+	// requires SNAT or not.
+	//	- 0b01: connection requires SNAT and is not marked with a ct mark.
```

Comment:
not sure  which one is correct but I suppose all `ct mark` string should be the same, either `CT mark` or `ct mark` in comments.

Reply:
Maybe CT mark.